cmd/commands.go

```diff
@@ -63,9 +62,8 @@ func (cmds *AppCommands) addCommand(cmd Command) {
 			if currentRegion != "" {
 				config.C.ScalingoRegion = currentRegion
 			}
-			action(c)
 
-			return nil
+			return action(c)
```

Comment:
With this change, if an action returns an error, it displays the error message (and that's it). There is no other messages than the error string representation.